docs/user/rocket/generic_surface.rst

```diff
@@ -0,0 +1,422 @@
+.. _genericsurfaces:
+
+Generic Surfaces and Custom Aerodynamic Coefficients
+====================================================
+
+Generic aerodynamic surfaces can be used to model aerodynamic forces based on 
+force and moment coefficients. The :class:`rocketpy.GenericSurface` class is a
+parent class for all generic surfaces. The :class:`rocketpy.LinearGenericSurface`
+class is a child class of :class:`rocketpy.GenericSurface` and is used to model
+aerodynamic forces based on linear force and moment coefficients.
+
+Both classes base their coeffcient on the definition of the aerodynamic frame
+of reference.
```

Comment:
can you mention why is this important? when can this kind of class be used?

I'd mention that it can make your simulation less dependent of barrowman coefficients